pkg/apis/machine/v1beta1/v1alpha1_suite_test.go

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
Just a nit.  `s/v1alpha1_suite_test.go/v1beta1_suite_test.go`. All just keep it as it is.